### PR TITLE
Advanced Search Tips

### PR DESCRIPTION
**Pull request by sephcoster**
Short description explaining the high-level reason for the pull request

## Additions

- Display component for Advanced Tips
- Method for SearchBar to hide / show Advanced Tips component

## Removals

- None

## Changes

- Enclosed SearchBar in a `<div>` so I could insert an additional `<nav>` element beneath the existing search bar.

## Testing

- 💯 

## Review

- @JeffreyMFarley @AdamZarger 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots
advancedtips

## Notes

-

## Todos

-

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/Dialogs/AdvancedTips.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import './AdvancedTips.less';
+import React from 'react';
+
+export default class AdvancedTips extends React.Component {
+
+  render() {
+    return (
+      <nav className="advanced_tips">
+          <div className="o-expandable">
+            <div className="o-expandable_content o-expandable_content__onload-open">
```
Comment (JeffreyMFarley) [inline src/Dialogs/AdvancedTips.jsx]: Glad you started with the Capital Framework styles

File: src/Dialogs/AdvancedTips.less [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    flex-basis: 31%;
+    padding-right: 2%;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    .example-input {
+      box-sizing: border-box;
+      width: 48%;
+      color: @gray-80;
+      display:inline-block;
+      &.first {
```
Comment (JeffreyMFarley) [inline src/Dialogs/AdvancedTips.less]: The other way to do this is [`input:first-of-type`](https://www.w3schools.com/cssref/sel_first-of-type.asp)

**Comment by sephcoster** [comment]
Thanks - will get to these issues shortly.

RE the linting alerts on having the ternary operator at the end of the line: Up-front seemed a bit more readable in the situation where you're mingling JSX with HTML. Changing it to follow the line now to remove the error, but what do you think moving forward? Is it clear with a scan where I'm using ternaries when they're at the end of the line?

**Comment by JeffreyMFarley** [comment]
Yeah, I'm not a fan of this linting rule, but I also don't like all the whitespace inside the parens

Basically, I am trying to use the CFPB style and not modify it significantly

**Pull request merged**